packages/material-ui/src/Select/Select.test.js

```diff
@@ -78,14 +78,14 @@ describe('<Select />', () => {
     );
   });
 
-  it('should have an input with [type="hidden"] by default', () => {
+  it('should have an input with [aria-hidden] by default', () => {
```

Comment:
What's the use case for this test? What about changing its purpose?
```suggestion
  it('should have an input with the value', () => {
```